docs/docs.md

```diff
@@ -29,6 +32,9 @@ Create a Dynamics client from environment variables:
 3. DYNAMICS_CLIENT_ID
 4. DYNAMICS_CLIENT_SECRET
 5. DYNAMICS_SCOPE
```

Comment:
See [this comment](https://github.com/MrThearMan/dynamics-client/pull/4/files#r1148501713).

Reply:
(I will abide by your preference)